File: C#/src/Controllers/UserController.cs
using Csharp.Core;

namespace Csharp.Controllers;

public class UserController
{
    public object Index()
    {
        Logger.Info("User list requested");

        var users = new[]
        {
            new { id = 1, name = "John" },
            new { id = 2, name = "Jane" },
            new { id = 3, name = "Mike" }
        };

        return new
        {
            status = "success",
            data = users
        };
    }
}

File: php/src/Controllers/UserController.php
<?php

namespace Volley5544\PhpProject\Controllers;

use Volley5544\PhpProject\Core\Logger;

class UserController
{
    public function index()
    {

        Logger::info("User list requested");

        $users = [
            ["id" => 1, "name" => "John"],
            ["id" => 2, "name" => "Jane"],
            ["id" => 3, "name" => "Mike"],
        ];

        return json_encode([
            "status" => "success",
            "data" => $users
        ]);
    }
}